src/main/java/it/pagopa/transactions/client/PaymentGatewayClient.java

```diff
@@ -619,7 +619,8 @@ public Mono<RedirectUrlResponseDto> requestRedirectUrlAuthorization(
                                     .paName(paName);// optional
                             Either<RedirectConfigurationException, URI> pspConfiguredUrl = getRedirectUrlForPsp(
                                     authorizationData.pspId(),
-                                    authorizationData.paymentTypeCode()
+                                    authorizationData.paymentTypeCode(),
+                                    authorizationData.pspChannelCode()
```

Comment:
```suggestion
                                    touchpoint
```

Use here touchpoint as key parameter (that can be one of CHECKOUT and IO as for now)

Reply:
Done.